src/test/java/org/folio/marc4ld/mapper/field650/Marc2Bibframe650IT.java

```diff
@@ -1,151 +1,148 @@
+        Map.entry("http://bibfra.me/vocab/marc/locationOfEvent", List.of("location of event")),
+        Map.entry("http://bibfra.me/vocab/marc/fieldLink", List.of("field link")),
+        Map.entry("http://bibfra.me/vocab/marc/source", List.of("source")),
+        Map.entry("http://bibfra.me/vocab/marc/controlField", List.of("control field")),
+        Map.entry("http://bibfra.me/vocab/lite/equivalent", List.of("equivalent")),
+        Map.entry("http://bibfra.me/vocab/marc/miscInfo", List.of("misc info")),
+        Map.entry("http://bibfra.me/vocab/lite/authorityLink", List.of("authority link")),
+        Map.entry("http://bibfra.me/vocab/lite/date", List.of("date")),
+        Map.entry("http://bibfra.me/vocab/marc/geographicCoverage", List.of("geographic coverage")),
+        Map.entry("http://bibfra.me/vocab/lite/name", List.of("name")),
+        Map.entry("http://bibfra.me/vocab/marc/materialsSpecified", List.of("materials specified")),
+        Map.entry("http://bibfra.me/vocab/marc/linkage", List.of("linkage"))
+      ),
+      "name"
+    );
+
+    // Validate subFocus resources
+    var subFocusEdges = getOutgoingEdges(subjectEdge, withPredicateUri(SUB_FOCUS.getUri()));
+    assertThat(subFocusEdges).hasSize(8);
+    var expectedSubFocuses = Map.of(
+      "form 1", new TypeAndProperties(FORM,
+        Map.of(
+          "http://bibfra.me/vocab/lite/label", List.of("form 1"),
+          "http://bibfra.me/vocab/lite/name", List.of("form 1"))
+      ),
+      "form 2", new TypeAndProperties(FORM,
+        Map.of(
+          "http://bibfra.me/vocab/lite/label", List.of("form 2"),
+          "http://bibfra.me/vocab/lite/name", List.of("form 2"))
+      ),
+      "topic 1", new TypeAndProperties(TOPIC,
+        Map.of(
+          "http://bibfra.me/vocab/lite/label", List.of("topic 1"),
+          "http://bibfra.me/vocab/lite/name", List.of("topic 1"))
+      ),
+      "topic 2", new TypeAndProperties(TOPIC,
+        Map.of(
+          "http://bibfra.me/vocab/lite/label", List.of("topic 2"),
+          "http://bibfra.me/vocab/lite/name", List.of("topic 2"))
+      ),
+      "temporal 1", new TypeAndProperties(TEMPORAL,
+        Map.of(
+          "http://bibfra.me/vocab/lite/label", List.of("temporal 1"),
+          "http://bibfra.me/vocab/lite/name", List.of("temporal 1"))
+      ),
+      "temporal 2", new TypeAndProperties(TEMPORAL,
+        Map.of(
+          "http://bibfra.me/vocab/lite/label", List.of("temporal 2"),
+          "http://bibfra.me/vocab/lite/name", List.of("temporal 2"))
+      ),
+      "place 1", new TypeAndProperties(PLACE,
+        Map.of(
+          "http://bibfra.me/vocab/lite/label", List.of("place 1"),
+          "http://bibfra.me/vocab/lite/name", List.of("place 1"))
+      ),
+      "place 2", new TypeAndProperties(PLACE,
+        Map.of(
+          "http://bibfra.me/vocab/lite/label", List.of("place 2"),
+          "http://bibfra.me/vocab/lite/name", List.of("place 2"))
+      )
+    );
+
+    subFocusEdges
```

Comment:
these lines of code along with ```TypaAndProperties``` record are being duplicated in number of such new separate tests
is it possible to move them into ```Marc2LdTestBace``` class?

Reply:
Good idea. Will do.